### PR TITLE
Update npm packages

### PR DESCRIPTION
**Pull request by htdat**
Close #620 
Close #629 
Close #640 
Close #641 

## Description

- Updated `node-sass` to `4.14.1` (similar to PR #620)
- Updated various npm packages by running `npm update` (including update for lodash #641) 
- #640 is not necessary as `ua-parser-js` no longer exists on any dependency after `npm update` above. 
- Tried to patch "Bump elliptic from 6.4.1 to 6.5.4  #629" 

## Steps to Test

Looking at the `E2E and JS tests` for this PR https://github.com/Automattic/Edit-Flow/runs/2562511361?check_suite_focus=true

### CODE REVIEW

**Comment by mikeyarce** [comment]
Thanks @hdat - Is there another way to test this without relying on GitHub Actions?  Just wondering if I could test this locally and what I should expect or be able to see.

**Comment by htdat** [comment]
@mikeyarce 

> Is there another way to test this without relying on GitHub Actions? Just wondering if I could test this locally and what I should expect or be able to see.

Yes, you can test this on your local machine but note that it's not exactly the same with GitHub Actions. 

## Steps to run the similar test on your local machine:

1. First, you will need to install NodeJS on your computer. It should use v14. E.g: in my computer, it looks like this:
```
~ node -v
v14.15.4
``` 
2. Make sure that you have Docker installed. Docker is used to set up test sites with [the `wp-env` package](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/). AFAIK, the Docker version does not matter. But here is what I have on my local machine: 
```
docker -v
Docker version 19.03.13, build 4484c46d9d
``` 
3. Follow all `npm` commands starting from this line: 
https://github.com/Automattic/Edit-Flow/blob/2642a915918e53b94f62d7ec11268b7165b498cc/.github/workflows/e2e-and-js-tests.yml#L28

For step 3, you can replace `npm ci` with `npm install` in your local machine but in general, that does not matter much as you see the info here https://docs.npmjs.com/cli/v7/commands/npm-ci

## What you can expect: 

- If you see errors when running `npm ci` or `npm install`, you can ignore them, as long as you do not see any error for `npm run build`.
- If you see errors with `npm run wp-env start`, it's likely an error with Docker. 
- You will not see errors for `Lint JS` and `JS tests` but you may see errors for `E2E tests`. 

Regarding E2E tests, I've found out that some tests in tests/e2e/specs/calendar-body.test.js are not stable (A new issue will be created soon).

**Review by nielslange — COMMENTED** [review]
Other than the different version of the dependencies, I can confirm that only `npm run test-e2e` shows an error when running locally. All other commands run as expected and this PR looks good to me.

File: package.json [hunk 1]
```diff
@@ -42,29 +42,29 @@
...
-    "@testing-library/user-event": "^10.0.2",
-    "@wordpress/components": "^9.4.1",
-    "@wordpress/e2e-test-utils": "^4.5.0",
-    "@wordpress/element": "^2.13.1",
-    "@wordpress/env": "^4.0.2",
-    "@wordpress/i18n": "^3.11.0",
-    "@wordpress/jest-console": "^3.6.0",
-    "@wordpress/jest-preset-default": "^6.0.0",
-    "@wordpress/jest-puppeteer-axe": "^1.7.0",
+    "@babel/core": "^7.14.0",
```
[the remark this answers is not in the document]
Comment (htdat) [reply package.json:45]: > Is there a specific reason, why you're not using the latest version, in this case 7.14.3, as seen on https://www.npmjs.com/package/@babel/core?

I simply used `npm update` for all of these packages as mentioned in this commit https://github.com/Automattic/Edit-Flow/pull/645/commits/f8c2ce15f13321fff350eed8fc9ed28f9d60153d

It looks like 7.14.3 has just been released after my PR here. https://www.npmjs.com/package/@babel/core

The same should go for other dependencies.

[the remark this answers is not in the document]
Comment (htdat) [reply package.json]: I've run `npm packages`. All of the packages should be up-to-date now.

**Review by mikeyarce — APPROVED** [review]

**Pull request merged**